CHANGELOG.md

```diff
@@ -15,7 +16,7 @@ Changelog
 [#217]: https://github.com/GIScience/ohsome-api/issues/217
 [#228]: https://github.com/GIScience/ohsome-api/pull/228
 [#230]: https://github.com/GIScience/ohsome-api/pull/230
-
+[#233]: https://github.com/GIScience/ohsome-api/issues/227
```

Comment:
let' use the ticket number for the markdown link: `[#227]` instead of ~`[#233]`~. This needs to be changed above in line 9 as well.

Reply:
My bad, I missaw the issue number.